### PR TITLE
Add Global Event stream timeline

### PR DESCRIPTION
**Pull request by viceromag**
In progress

### CODE REVIEW

File: spec/definitions/Timelines/GlobalTimeline.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      - subscription-was-activated
+      - subscription-was-canceled
+      - subscription-was-renewed
+      - subscription-was-upgraded
+      - subscription-was-downgraded
+  data:
+      type: object
+      example:
+        "transactionId": "test-transaction"
+        "amount": 10
```
Comment (vova07) [inline spec/definitions/Timelines/GlobalTimeline.yaml]: It will be `10.00` instead.

File: spec/definitions/Timelines/GlobalTimeline.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      - subscription-trial-was-ended
+      - subscription-was-activated
+      - subscription-was-canceled
+      - subscription-was-renewed
+      - subscription-was-upgraded
+      - subscription-was-downgraded
+  data:
+      type: object
+      example:
+        "transactionId": "test-transaction"
```
Comment (slavcodev) [inline spec/definitions/Timelines/GlobalTimeline.yaml]: Are you going to define every event separately? Using discriminator?

File: spec/paths/timelines@global.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+get:
+  tags:
+    - Timelines
+  summary: Retrieve a list of events in Global timeline
+  parameters:
```
Comment (adamaltman) [inline spec/paths/timelines@global.yaml]: What filters are available?  

Is it clear that it is always sorted by `occurredTime` descending?

**Comment by adamaltman** [comment]
This seems like it would fit better in this repo: https://github.com/rebilly/RebillyReportsAPI/

Aren't these a kind of report?  eg. read only data...

File: spec/definitions/Timelines/GlobalTimeline.yaml (hunk 2 above)
Comment (viceromag) [reply spec/definitions/Timelines/GlobalTimeline.yaml]: Each event might contain different attributes in `data`

Comment (slavcodev) [reply spec/definitions/Timelines/GlobalTimeline.yaml]: Yes, and this is the reason why I asked about discriminator.

Comment (viceromag) [reply spec/definitions/Timelines/GlobalTimeline.yaml]: `type` will be a discriminator in that case.

Comment (slavcodev) [reply spec/definitions/Timelines/GlobalTimeline.yaml]: So what is answer to my question? Are you going to define every timeline event with concrete scheme for `data`?

**Comment by viceromag** [comment]
Closing this PR, moving endpoint to ReportsAPI

**Comment by pmeth** [comment]
https://github.com/Rebilly/RebillyReportsAPI/pull/50

**Pull request closed without merging**